Resources/public/js/views/services/plugins/ez-objectrelationsloadplugin.js

```diff
@@ -79,6 +80,60 @@ YUI.add('ez-objectrelationsloadplugin', function (Y) {
         },
 
         /**
+         * Loads the related contents of the fields. Once this is done, it sets the contents in
+         * the `relatedContents` attribute of the event target.
+         *
+         * If `loadLocation` or `loadLocationPath` is TRUE, then array of content structs is being set in
+         * the `relatedContents` attribute of the event target. Content struct contains `content` (eZ.Content)
+         * and `location` (eZ.Location) attributes.
+         *
+         * @protected
+         * @method _loadFieldRelations
+         * @param {Object} e FieldRelations event facade
+         * @param {Bool} e.loadLocation flag indicating whether to load the Location
+         * @param {Bool} e.loadLocationPath flag indicating whether to load the Location's path. If it is TRUE it
+         * forces to e.loadLocation be also TRUE
+         */
+        _loadFieldRelations: function (e) {
+            var relatedContentListArray = [],
+                stack = new Y.Parallel(),
+                loadedRelation = {},
+                loadingError = false,
+                fields = this.get('host').get('content').get('fields'),
+                destinationContentIds = fields[e.fieldDefinitionIdentifier].fieldValue.destinationContentIds,
+                end = stack.add(function (error, struct) {
+                    if (error) {
+                        e.target.set("loadingError", true);
+                        loadingError = true;
+                    } else {
+                        relatedContentListArray.push(struct);
+                    }
+                });
+
+            Y.Array.each(destinationContentIds, function (contentId) {
```

Comment:
actually the problem lies in eZ.Content#relations where the field value should be used when we want the relations of type `ATTRIBUTE`